src/idvc/dvc_interface.py

```diff
@@ -2052,6 +2052,7 @@ def extendMask(self, **kwargs):
 
         # save the mask to a file in temp folder
         writer = vtk.vtkMetaImageWriter()
+        writer.SetCompression(True)
```

Comment:
This is in the hope that writing will be faster. The idea is that the mask is sparse and it will be quick to zip.

Reply:
compression is True by default anyway

Reply:
it didn't make any difference in fact!